chat/indico_chat/templates/manage_event.html

```diff
@@ -1,7 +1,5 @@
-{% extends 'layout/base.html' %}
 {% from 'forms/_form.html' import form_header, form_row, form_footer  %}
 
-{% block title %}{% trans %}Chat Rooms{% endtrans %}{% endblock %}
 {% block content %}
```

Comment:
While not absolutely necessary (the block simply doesn't do anything if no inheritance is used), it would be cleaner to remove this block tag too and outdent its contents